src/test/java/io/vertx/test/core/BlockedThreadCheckerTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2014 Red Hat, Inc.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Apache License v2.0 which accompanies this distribution.
+ *
+ * The Eclipse Public License is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * The Apache License v2.0 is available at
+ * http://www.opensource.org/licenses/apache2.0.php
+ *
+ * You may elect to redistribute this code under either of these licenses.
+ */
+
+package io.vertx.test.core;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Verticle;
+import io.vertx.core.Vertx;
+import io.vertx.core.VertxOptions;
+
+import org.junit.Test;
+
+/**
+ * @author <a href="http://tfox.org">Tim Fox</a>
+ */
+public class BlockedThreadCheckerTest extends VertxTestBase {
```

Comment:
main reason for the test was to get 100% coverage, actually asserting the logging calls is a bit difficult since it requires a log mock of some kind